server/src/main/java/io/littlehorse/server/streamsimpl/storeinternals/GetableStorageManager.java

```diff
@@ -17,7 +17,7 @@
 @Slf4j
 public class GetableStorageManager {
 
-    private final Map<String, StoredGetable<? extends Message, ? extends Getable<?>>> uncommittedChanges;
+    private final Map<String, StoredGetable<?, ?>> uncommittedChanges;
```

Comment:
Yeah, you are right `Map<String, StoredGetable<? extends Message, ? extends Getable<?>>>` is probably redundant considering that StoredGetable already specifies the types it can receive.